tests/integration/framework/process/daprd/daprd.go

```diff
@@ -329,7 +329,7 @@ func (d *Daprd) ProfilePort() int {
 }
 
 // Metrics Returns a subset of metrics scraped from the metrics endpoint
-func (d *Daprd) Metrics(t *testing.T, ctx context.Context) map[string]float64 {
+func (d *Daprd) Metrics(t *testing.T, ctx context.Context, desiredNameAndLabelPairs ...string) map[string]float64 {
```

Comment:
@jake-engelberg  Take a look at this PR, seems to be relevant here. https://github.com/dapr/dapr/pull/8218
@JoshVanL can we merge that please to unblock this PR?